madlibs/src/Story.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+// basic story
+const Story = ({verb='fighting',noun='snakes', adjective='god-forsaken'}) => {
+
+    return(
+        <>
+        <div>
+        <p>"There are too many {verb}</p>
+        <p>{noun} on this </p>
+        <p>{adjective} plane!" he screamed</p>
+        </div>

```

Comment:
Although whitespace doesn't always directly impact html, best to have things indented properly to see hierarchy clearly
```suggestion
        <div>
            <p>"There are too many {verb}</p>
            <p>{noun} on this </p>
            <p>{adjective} plane!" he screamed</p>
        </div>

```